mapie/tests/test_classification.py

```diff
@@ -821,6 +822,18 @@
     random_state=random_state,
 )
 
+gb = GradientBoostingClassifier(
+            random_state=random_state
+            )
```

Comment:
```suggestion
gb = GradientBoostingClassifier(random_state=random_state)
```

Reply:
I also suggest moving it directly into the test (before instantiating the `mapie` estimator) to better understand where `gb` is being used.